scripts/background.js

```diff
@@ -491,50 +491,50 @@ chrome.runtime.onMessage.addListener(function(message,sender,sendResponse){
         }else if(message.message=='showall'){
           chrome.storage.sync.get(['show_context'],function(event){
             if(!event.show_context){
-              event.show_context="tab";
+              event.show_context="tab"; //By-default the context-window open in tabs
             }
+            var received_url=message.url; //URL which is received by message-parsing
+            var length=received_url.length; 
+            var start_index="";
+            if(received_url.includes('https://')){
```

Comment:
Again, please use `url = url.replace(/^https?:\/\//,'')` to strip `http://` and `https://`` prefixes. Please update this code as well.